codebuild_specs/mock_e2e_tests.yml

```diff
@@ -0,0 +1,10 @@
+version: 0.2
+env:
+  shell: bash
+  variables:
+    JEST_JUNIT_OUTPUT: "reports/junit/js-test-results.xml"
+    NODE_OPTIONS: --max-old-space-size=4096
```

Comment:
This is to increase maximum size of the garbage collected heap in case of crash due to exceeding. The options are also used in circleci.
FYI: https://stackoverflow.com/questions/48387040/how-do-i-determine-the-correct-max-old-space-size-for-node-js